KanvasExample/KanvasExampleTests/Editor/MultiEditor/MultiEditorControllerTests.swift

```diff
@@ -164,7 +165,8 @@ final class MultiEditorControllerDelegateStub: MultiEditorComposerDelegate {
     }
 
     func editor(segment: CameraSegment, edit: EditorViewController.Edit?) -> EditorViewController {
-        return EditorViewController(settings: settings,
+        return EditorViewController(delegate: editorDelegate,
```

Comment:
It looks like this is breaking the unit tests right now since we're not going to pass an `EditorControllerDelegate` in the initializer anymore.